model/database/spans.yaml

```diff
@@ -254,14 +254,14 @@ groups:
         examples: ["mykeyspace"]
         requirement_level:
           conditionally_required: If available.
-      - ref: db.cassandra.page_size
-      - ref: db.cassandra.consistency_level
+      - ref: cassandra.page_size
+      - ref: cassandra.consistency.level
```

Comment:
are there any other keys for `consistency` besides `level`? 
the impression I had was that we used `.` when there are other possible keys, so I was just curious on why this one changed to `consistency.level`, since I don't see other keys being used for it (unless I missed it)

For example, the one right above kept as `page_size`, so wondering why just this one changed

Reply:
Great point and good test for our recent additions to the naming guidance:

https://github.com/open-telemetry/semantic-conventions/blob/8d9d4a1ce84b060e4d13b5f44b3495648767deb5/docs/general/naming.md?plain=1#L57-L68

1. `consistency.level` makes sense to me 
2.  I can come up with an artificial example of `consistency.quorum` - I had to think for a while though. 

So based on this guidance, it *could* be `.`, so it should be `.`.

I applied the same litmus test to page_size and it also makes sense plus I can come up with some examples like `page.number`, `page.next_link`. So I changed it all to `page.size`.

Really curious what do you think about this new guidance

Reply:
I like the new guidance! I was more curious because I saw `page_size` and `consistency_level` one right after the other but they were treated differently. 
Now with both using `.`, I'm happy 😄